src/components/TodoList.js

```diff
@@ -0,0 +1,27 @@
+import TodoItem from './TodoItem';
+
+const TodoList = () => {
+  const getTodo = [
+    { id: 123, isCompleted: false, todo: 'todotodo', userId: 111 },
+    { id: 124, isCompleted: true, todo: 'todo', userId: 111 },
+    { id: 125, isCompleted: false, todo: 'todoto', userId: 111 },
+  ];
+
+  return (
+    <>
+      {getTodo?.map((todo) => {
```

Comment:
```suggestion
      {getTodo?.map(({id,todo,isCompleted}) => {
```
로 바꿔주시면 태그에 todo로 시작하는 부분들을 지울 수 있을 것 같아요